converters/toZigbee.js

```diff
@@ -3129,7 +3129,7 @@ const converters = {
         },
     },
     moes_thermostat_mode: {
-        key: ['preset'],
+        key: ['preset_mode'], // to make it identical with fromZigbee
```

Comment:
please change this to `key: ['preset', 'preset_mode'],` to prevent a breaking change

Reply:
> please change this to `key: ['preset', 'preset_mode'],` to prevent a breaking change

Make sense. Done